bluebrain/repo-bluebrain/packages/py-cwl-registry/package.py

```diff
@@ -10,12 +10,12 @@ class PyCwlRegistry(PythonPackage):
     git = "ssh://git@bbpgitlab.epfl.ch/nse/cwl-registry.git"
 
     version("develop", branch="main")
-    version("0.4.4", tag="cwl-registry-v0.4.4")
+    version("0.4.6.dev2", tag="cwl-registry-v0.4.6.dev2")
```

Comment:
This lead to bad results the last time around.

Reply:
Thanks for bringing this up. I removed the dev2 to test the 0.4.6.dev2 as "0.4.6".